README.MD

```diff
@@ -202,7 +148,28 @@ the setup was also tested to work with Gigabyte NUC, Intel Edison, BeagleBone
 Black and Intel Galileo V2 but they might require some work on the BeagleBone
 filesystem to make them work.
 
-## 2.2 Setting up BeagleBone Black filesystem on the host PC
+## 2.2 BBB booting
+
+This section will give you an overview of the BBB booting components, boot sequence and how it relates to the PC Host. This should give you a good understanding of why we need to setup various different services and what is their role.
+
+![](readme_pictures/BBB_booting.png)
+
+_BeagleBone Black booting with the PC Host._
```

Comment:
The picture and the picture text would look nicer if the image didn't have so much white space at the bottom. Not a big deal though.